### ISSUE TITLE
bad utf-8 encoding when reading a pem certificate

### ISSUE THREAD

**Issue by iamtxena** [kjur/jsrsasign#473]
Hello, I am trying to generate a CADES signature and my code is working when using a x509 certificate with no special characters. However, when using one that contains accents in the issuer attributes. It fails the verification because the issuer name does not match.

This is because the resulting issuer text encoding is not properly done.

I attach the given certificate that fails. Just rename it as `.pem`
[certificate.txt](https://github.com/kjur/jsrsasign/files/6036163/certificate.txt)

When using `openssl` to read this certificate:
```sh
$ openssl x509 -noout -text -in tests/data/certificate.pem

Certificate:
    Data:
        Version: 3 (0x2)
        Serial Number:
            2e:0c:c0:ec:99:62:a3:a4:38:b5:92:88:fa:87:da:a7:21:ea:24:88
        Signature Algorithm: sha256WithRSAEncryption
        Issuer: C = ES, ST = CAT, L = Barcelona, O = Internet Widgits Pty Ltd, OU = ValidatedId, CN = C\C3\A0nary
...
```
As you can see the CN contains two UTF-8 characters `\C3\A0` to represent the `à` character.

When reading the pem certificate using this library ( `"jsrsasign": "^10.1.11"`) and printing the issuer I got it wrong:
```js
const x509KjurCert = new X509();
x509KjurCert.readCertPEM(pemCert);
console.log(x509KjurCert.getIssuerString());
// /C=ES/ST=CAT/L=Barcelona/O=Internet Widgits Pty Ltd/OU=ValidatedId/CN=CÃ nary
```

And when using the node crypto v15.10 it prints the expected well coded string:
```js
const x509cert = new X509Certificate(pemCert);
const { issuer } = x509cert;
const finalIssuer = `/${issuer.replace(/\n/g, "/") as string}`;
console.log(finalIssuer);
// /C=ES/ST=CAT/L=Barcelona/O=Internet Widgits Pty Ltd/OU=ValidatedId/CN=Cànary
```
To pass the validation, I have to use the issuer parsed from node crypto.

Is there a way to force this library to code correctly the given string as UTF-8?

Thanks,
Albert

**Comment by kjur** [comment]
Hi @iamtxena , I found the cause of issue. I'll fix this in the next version. Thanks.

**Comment by kjur** [comment]
Hi @iamtxena , this issue was fixed in the 10.1.12 release today. This ticket will be closed. Thanks.

**Comment by iamtxena** [comment]
Working on this new `10.1.12` release. 

Thanks for the quick response! Much appreciated.
